bindings/python/pydeck/pydeck/bindings/layer.py

```diff
@@ -72,25 +33,128 @@ def __init__(
         elevation_range=None,
         elevation_scale=None,
         extruded=None,
-        upper_percentile=100,
+        upper_percentile=None,
```

Comment:
Will it make sense to automatically generate this list from each layer's `defaultProps` list?

Reply:
It seems to me that the best approach would be for the pydeck layer to not specify defaults, and instead just omit props when generating the JSON payload if they were not specified in python. 

That way the default prop values implemented in deck.gl (and described in deck.gl docs) will always take effect.

Since different layers could potentially have different defaults for a similarly named prop, trying to collect a superset of all props here might not work.

Maybe setting default to `None` here and not adding them to JSON payload in that case?

Reply:
If `None` is passed, the default within deck.gl is actually used. The `layer` object in this file is converted to JSON, and if a property doesn't have a specified value, it doesn't get passed into the deck object.